frontend/pages/sign-out.tsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
```

Comment:
I know this is not a part of the task, but I needed this functionality to test the changes on the menu